doctr/__main__.py

```diff
@@ -193,6 +198,11 @@ def get_parser(config=None):
     configure_parser.set_defaults(func=configure)
     configure_parser.add_argument('--force', action='store_true', help="""Run the configure command even
     if we appear to be on Travis.""")
+    configure_parser.add_argument('--dkenv', type=str, metavar="ENVVAR",
```

Comment:
We should use the existing environment variable names. If this works we should use it by default and have the old way behind a flag (or just remove it).

Reply:
If you would be happy making the deploy key in an environment variable the default, then maybe you are right - re-using the old environment variable names would make sense.

Reply:
Assuming there are no issues with it we should. We will have to continue to support the old way at least for deploy for existing repos. It will need a lot of testing to make sure, once we get it working.